apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -94,38 +109,58 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
                 )
                 .flatMap((interestGroup) => interestGroup?.sys.id) || []),
             ]),
-            workingGroupLeadershipRoleCount: getUniqueIdCount(
-              team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                (teamMembershipItem) =>
-                  teamMembershipItem?.linkedFrom?.usersCollection?.items
-                    .filter(filterOutAlumni)
-                    .flatMap(flattenWorkingGroupLeaders),
-              ) || [],
-            ),
-            workingGroupMemberCount: getUniqueIdCount(
-              team.linkedFrom?.teamMembershipCollection?.items.flatMap(
-                (teamMembershipItem) =>
-                  teamMembershipItem?.linkedFrom?.usersCollection?.items
-                    .filter(filterOutAlumni)
-                    .flatMap(flattenWorkingGroupMember),
-              ) || [],
-            ),
-            workingGroupPreviousLeadershipRoleCount: getUniqueIdCount(
-              team.linkedFrom?.teamMembershipCollection?.items.flatMap(
+            workingGroupLeadershipRoleCount: team.inactiveSince
+              ? 0
+              : getUniqueIdCount(
+                  team.linkedFrom?.teamMembershipCollection?.items.flatMap(
+                    (teamMembershipItem) =>
+                      teamMembershipItem?.linkedFrom?.usersCollection?.items
+                        .filter(filterOutAlumni)
+                        .flatMap(flattenWorkingGroupLeaders),
+                  ) || [],
+                ),
+            workingGroupMemberCount: team.inactiveSince
+              ? 0
+              : getUniqueIdCount(
+                  team.linkedFrom?.teamMembershipCollection?.items.flatMap(
+                    (teamMembershipItem) =>
+                      teamMembershipItem?.linkedFrom?.usersCollection?.items
+                        .filter(filterOutAlumni)
+                        .flatMap(flattenWorkingGroupMember),
+                  ) || [],
+                ),
+            workingGroupPreviousLeadershipRoleCount: getUniqueIdCount([
+              ...((team.inactiveSince &&
+                team.linkedFrom?.teamMembershipCollection?.items.flatMap(
+                  (teamMembershipItem) =>
+                    teamMembershipItem?.linkedFrom?.usersCollection?.items
+                      .filter(filterOutAlumni)
+                      .flatMap(flattenWorkingGroupMember),
```

Comment:
Here, we're checking `workingGroupPreviousLeadershipRoleCount`, right? so we should flatten working group leaders instead of members?

```suggestion
                      .flatMap(flattenWorkingGroupLeaders),
```